app/Builder.zig

```diff
@@ -88,48 +84,42 @@ pub fn run(self: *Builder) !void {
             const conn = try server.accept();
             try pool.spawn(handleConn, .{ self, conn });
         }
-    } else {
-        try self.exec();
     }
 }
 
-fn exec(self: Builder) !void {
-    var arena = std.heap.ArenaAllocator.init(allocator);
-    defer arena.deinit();
-    const argv = try self.buildArgs(arena.allocator());
+fn runZigBuild(self: Builder) !void {
+    var args_arena = std.heap.ArenaAllocator.init(allocator);
+    defer args_arena.deinit();
+
+    const args = try self.buildArgs(args_arena.allocator());
 
-    return std.os.execvpeZ(
-        argv[0].?,
-        @ptrCast([*:null]const ?[*:0]const u8, argv),
-        @ptrCast([*:null]const ?[*:0]const u8, std.os.environ.ptr),
-    );
+    var child = std.ChildProcess.init(args, allocator);
+    _ = try child.spawnAndWait();
```

Comment:
doesn't check exit status

Reply:
True 😅 I guess it should just exit if the result is == .Signal / .Stopped / .Unknown too?

Reply:
exactly